hbase_storage/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java

```diff
@@ -54,9 +54,15 @@ public HPartitionCreator(HBaseTableInfoFactory tableInfoFactory,Connection conne
 
     @Override
     public PartitionCreator withName(String name){
+        return withName(name, 0);
+    }
+
+    @Override
+    public PartitionCreator withName(String name, int priority){
```

Comment:
Define a separate withPriority instead of conflating name and priority setters in one method.